apps/aecore/src/aec_tx_processor.erl

```diff
@@ -1494,7 +1805,11 @@ cache_write_through_fun({account,_Pubkey}, Account, Trees) ->
     ATrees  = aec_trees:accounts(Trees),
     ATrees1 = aec_accounts_trees:enter(Account, ATrees),
     aec_trees:set_accounts(Trees, ATrees1);
-cache_write_through_fun({call,_Id}, Call, Trees) ->
+cache_write_through_fun({auth_call, _Id}, Call, Trees) ->
+    CTree  = aec_trees:calls(Trees),
+    CTree1 = aect_call_state_tree:enter_auth_call(Call, CTree),
+    aec_trees:set_calls(Trees, CTree1);
```

Comment:
Changes in this file may conflict with https://github.com/aeternity/aeternity/pull/2319

Reply:
Yes, Tobias knows about this 👍